src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1106,10 +1106,10 @@ private void Open(object parameters)
         {
             // try catch for exceptions thrown while opening files, say from a future version, 
             // that can't be handled reliably
+            string xmlFilePath = string.Empty;
+            bool forceManualMode = false; 
             try
             {
-                string xmlFilePath = string.Empty;
-                bool forceManualMode = false;
```

Comment:
hmm.. For system default variable declaration, I see no point of putting it in the try block. What do you think?